### PR TITLE
Support the Source1/Source2 buttons, Vinyl passthrough

### PR DESCRIPTION
**Pull request by kaie**
Add support for the Source1 and Source2 buttons on the Hercules DJ Console RMX.
Map the buttons to toggle Vinyl Passthrough and set the button status LEDs.

### CODE REVIEW

**Comment by ywwg** [comment]
Hello!  this is a useful change, thank you.  Have you signed our contributor agreement?  We need you to do so before we merge this in.  https://docs.google.com/forms/d/1m_Ql2LyQ2nsgpFe6S4ED3jmzaLgA3U6P2TALOHMxGxA/viewform

**Comment by kaie** [comment]
On Wed, 2015-05-27 at 08:33 -0700, Owen Williams wrote:
> Hello!  this is a useful change, thank you.  Have you signed our contributor agreement?  We need you to do so before we merge this in.  https://docs.google.com/forms/d/1m_Ql2LyQ2nsgpFe6S4ED3jmzaLgA3U6P2TALOHMxGxA/viewform

signed

File: res/controllers/Hercules-DJ-Console-RMX-scripts.js [hunk 1]
```diff
@@ -450,6 +450,19 @@ HerculesRMX.Deck.prototype.cueSelectHandler = function(value) {
    }
 };
 
+HerculesRMX.Deck.prototype.sourceSelectHandler = function(value) {
```
Comment (ywwg) [inline res/controllers/Hercules-DJ-Console-RMX-scripts.js]: is there a reason this has to be a script binding and not just a basic toggle button?

Comment (kaie) [inline res/controllers/Hercules-DJ-Console-RMX-scripts.js]: On Wed, 2015-05-27 at 08:56 -0700, Owen Williams wrote:
> > +HerculesRMX.Deck.prototype.sourceSelectHandler = function(value) {
> 
> is there a reason this has to be a script binding and not just a basic toggle button?

I have no idea, I was happy that I was able to make it work.

Comment (kaie) [inline res/controllers/Hercules-DJ-Console-RMX-scripts.js]: If you can point me to an example button that makes use of the approach you're suggesting, I could try to test it later.

Comment (ywwg) [inline res/controllers/Hercules-DJ-Console-RMX-scripts.js]: take a look at how Vestax 400 does pfl, for instance.  Notice that there is a binding for the control itself (which was learned through the midi wizard) and an output for turning on the light.

**Comment by rryan** [comment]
What's the status here? Any reason not to merge as is?

**Comment by kaie** [comment]
sorry, I didn't have the time to follow up, and I don't expect to soon. If you could merge as is, that would be great.

**Comment by rryan** [comment]
thank you @kaie !

**Pull request merged**